panel/chat/feed.py

```diff
@@ -461,7 +458,7 @@ def send(
         self,
         value: ChatMessage | dict | Any,
         user: str | None = None,
-        avatar: str | BinaryIO | None = None,
+        avatar: str | BytesIO | None = None,
```

Comment:
should all of these type declarations include `bytes` as well?

Reply:
Yup! Thanks for catching that. I just pushed another commit